_includes/syllabus.html

```diff
@@ -33,12 +38,14 @@ <h2 id="schedule">Schedule</h2>
           <td class="col-md-7"></td>
         </tr>
       {% endif %}
-      {% assign current = site.start_time %} {% comment %}Re-set start time of this episode to general daily start time {% endcomment %}
+      {% if partnotday %}{% else %}
+        {% assign current = site.start_time %} {% comment %}Re-set start time of this episode to general daily start time {% endcomment %}
+      {% endif %}
     {% endif %}
     {% assign hours = current | divided_by: 60 %}
     {% assign minutes = current | modulo: 60 %}
     <tr>
-      {% if multiday %}<td class="col-md-1">{% if episode.start %}Day {{ day }}{% endif %}</td>{% endif %}
+      {% if multiday %}<td class="col-md-1">{% if episode.start %}{% if partnotday %}Part{% else %}Day{% endif %} {{ day }}{% endif %}</td>{% endif %}
```

Comment:
The condition here looks wrong. Would you not need

~~~
{% if multiday %}...{% endif %}
{% if partnotday %}...{% endif %}
~~~

Reply:
I think it should as follows: 
```
{% if multiday %}<td class="col-md-1">{% if episode.start %}Day {{ day }}{% endif %}</td>{% endif %}
{% if multipart %}<td class="col-md-1">{% if episode.part %}Part {{ part }}{% endif %}</td>{% endif %}
```

Note, I used `multipart` rather than `partnotday` and non-existent (at the moment) `part` variable